CONTRIBUTING.md

```diff
@@ -109,115 +106,72 @@ If at any time you want to blow away the tox created virtual environments and st
-```
-
-To test a package being developed against latest dev build version of dependent packages:
-a. cd to package root folder
-b. run tox environment devtest
+tox run -e whl -c <path to tox.ini
 
-```
-\> tox run -e devtest -c <path to tox.ini> --root <path to python package>
+> --root <path to python package> -- -p pdb
 ```
 
-This tox test( devtest) will fail if installed dependent packages are not dev build version.
-
-## Samples
-
-### Third-party dependencies
-
-Third party libraries should only be included in samples when necessary to demonstrate usage of an Azure SDK package; they should not be suggested or endorsed as alternatives to the Azure SDK.
-
-When code samples take dependencies, readers should be able to use the material without significant license burden or research on terms. This goal requires restricting dependencies to certain types of open source or commercial licenses.
+### Performance Testing
 
-Samples may take the following categories of dependencies:
+For full details on this framework, and how to write and run tests for an SDK, see the [perfstress tests documentation](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/metrics/tests/perfstress/README.md).
 
-- **Open-source** : Open source offerings that use an [Open Source Initiative (OSI) approved license](https://opensource.org/licenses). Any component whose license isn't OSI-approved is considered a commercial offering. Prefer OSS projects that are members of any of the [OSS foundations that Microsoft is part of](https://opensource.microsoft.com/ecosystem/). Prefer permissive licenses for libraries, like [MIT](https://opensource.org/license/MIT) and [Apache 2](https://opensource.org/license/apache-2-0). Copy-left licenses like [GPLv3](https://opensource.org/license/gpl-3-0) are acceptable for tools, and OSs. [Kubernetes](https://github.com/kubernetes/kubernetes), [Linux](https://github.com/torvalds/linux), and [Newtonsoft.Json](https://github.com/JamesNK/Newtonsoft.Json) are examples of this license type. Links to open source components should be to where the source is hosted, including any applicable license, such as a GitHub repository (or similar).
+### More Reading
 
-- **Commercial**: Commercial offerings that enable readers to learn from our content without unnecessary extra costs. Typically, the offering has some form of a community edition, or a free trial sufficient for its use in content. A commercial license may be a form of dual-license, or tiered license. Links to commercial components should be to the commercial site for the software, even if the source software is hosted publicly on GitHub (or similar).
+We maintain an [additional document](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/metrics/README.md) that contains detailed information regarding what is actually happening in these executions.
```

Comment:
It seems like this link was updated incorrectly. It should point to https://github.com/Azure/azure-sdk-for-python/blob/main/doc/eng_sys_checks.md